src/redux/books/booksSlice.js

```diff
@@ -1,52 +1,80 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const baseUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+const key = '4sREW0WX96Co7j4nt5il';
```

Comment:
[OPTIONAL]

I think it will be better if you dont expose your API key (this is a secret key) on Github to prevent any malicious activity on your codebase 👍🏽
It's always best to store your secret keys to environment variables so that they won't be exposed on GitHub. This way, your app will be more secure 😄

read about environment variables [here](https://create-react-app.dev/docs/adding-custom-environment-variables/)

Reply:
@vickymarz Thank you so much for your kind words and appreciation as well as for bringing up this important issue.